twake/backend/node/src/services/channels/entities/channel-activity.ts

```diff
@@ -11,17 +11,17 @@ import { ChannelType } from "../types";
 export class ChannelActivity {
   // uuid-v4
   @Type(() => String)
-  @Column("company_id", "string", { generator: "uuid" })
+  @Column("company_id", "plainstring", { generator: "uuid" })
```

Comment:
instead of having to change every `string` to `plainstring`, why don't you create a new type like `encodedstring` ?

Reply:
Romaric told me to do that, we can talk about this tomorrow during the daily.